src/Stratis.Features.Collateral/CheckCollateralFullValidationRule.cs

```diff
@@ -53,7 +50,15 @@ public override Task RunAsync(RuleContext context)
 
             IFederationMember federationMember = this.slotsManager.GetFederationMemberForTimestamp(context.ValidationContext.BlockToValidate.Header.Time);
 
-            byte[] rawCommitmentData = this.encoder.ExtractRawCommitmentData(context.ValidationContext.BlockToValidate.Transactions.First());
+            var commitmentHeightEncoder = new CollateralHeightCommitmentEncoder(this.Logger);
+
+            // Log each transaction for debug purposes.
+            foreach (Transaction transaction in context.ValidationContext.BlockToValidate.Transactions)
+            {
+                this.Logger.LogDebug(transaction.ToString());
```

Comment:
Hmm, is this just going to log the txid?

Reply:
Ah yes I need to add `network`